templates/layout.html

```diff
@@ -210,7 +118,7 @@
   </footer>
 </template>
 
-<template name="_loginButtonsAdditionalLoggedInDropdownActions">
+<template name="_loginButtonsAdditionalLoggedInDropdownActions"> //TODO This is unused.
```

Comment:
You're right `//` comments are not respected, and the comment shows up on the page.

I try to live by the rule that you need to be able to find usages of something by searching for the name. You devolve into "write only code" pretty quick otherwise.

I put in a comment.